spynnaker/pyNN/models/neuron/abstract_population_vertex.py

```diff
@@ -97,7 +95,7 @@ class AbstractPopulationVertex(
     # 5 elements before the start of global parameters
     # 1. has key, 2. key, 3. n atoms,
     # 4. n synapse types, 5. incoming spike buffer size.
-    _BYTES_TILL_START_OF_GLOBAL_PARAMETERS = 5 * BYTES_PER_WORD
+    BYTES_TILL_START_OF_GLOBAL_PARAMETERS = 5 * BYTES_PER_WORD
```

Comment:
I think this should be in the Machine vertex, as it is a property of that really.

Reply:
its a property of the neuron impl, which is still inside the app vertex, as its data related, it was deemed to stay there so that olivers stuff could use it without clones

Reply:
OK fair enough